documentation/src/main/asciidoc/topics/proc_setting_persistent_location.adoc

```diff
@@ -40,8 +37,33 @@ public GlobalStateConfigurationBuilder persistentLocation(String path, String re
 }
 ----
 
-{brandname} logs the following message if you attempt to use a directory
-outside the global persistent location with file-based cache stores:
+.File-Based Cache Stores and Global Persistent Location
+
+When using file-based cache stores, you can optionally specify filesystem directories for storage. These paths are relative to the global persistent location.
```

Comment:
"When using file-based cache stores, you can optionally specify filesystem directories for storage. Unless absolute paths are declared, these are always relative to the global persistent location." ?